### ISSUE TITLE
Top progress bar color - would we consider teal instead of blue? 

### ISSUE THREAD

**Issue by hwikyounglee** [GoogleChrome/web.dev#384]
🤔wondering if we would try the teal color in the logo. Hoping we use Blue for CTA and selected state etc as an accent color, I suggest we use something complementary for progress bar that's shown frequently e.g. page load.

```
body[theme="web-theme"] #app-progress .devsite-progress--indeterminate .devsite-progress--indeterminate-1,body[theme="web-theme"] #app-progress .devsite-progress--indeterminate .devsite-progress--indeterminate-3 {
    /*background-color: #3740ff*/
    background-color: #00C9DB
}
```

**Comment by robdodson** [comment]
Sent in a CL to make this configurable. I should be able to follow up once that lands with a fix for this.

**Comment by robdodson** [comment]
Fixed in cl/226353371
